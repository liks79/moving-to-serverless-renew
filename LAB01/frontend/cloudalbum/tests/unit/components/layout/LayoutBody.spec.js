import { shallowMount } from '@vue/test-utils';
import LayoutBody from '@/components/layout/LayoutBody.vue';

describe('LayoutBody', () => {
  const wrapper = shallowMount(LayoutBody);

  it('renders router', () => {
    expect(wrapper.find('router-view-stub').exists()).toMatchSnapshot();
  });

  it('renders v-content', () => {
    expect(wrapper.find('v-content-stub').exists()).toBe(true);
  });
});
