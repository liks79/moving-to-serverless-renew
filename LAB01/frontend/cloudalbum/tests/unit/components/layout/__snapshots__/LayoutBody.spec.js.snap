// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LayoutBody renders router 1`] = `true`;
